### DISCUSSION TITLE
Videos I upload into repository issues do not play

### DISCUSSION THREAD

**Discussion by reportthings** [community/community#31905] [Other Feature Feedback, Questions, & Ideas]
https://user-images.githubusercontent.com/85461551/188301028-729591e5-818e-484b-a04f-e6c2834afd41.mp4

Here is one example. The last 3 times I have uploaded a screen recording to showcase an issue, GitHub fails to play it.  The video processes, preview works, shows the length, but does not play.

I've tried searching for similar but I continually get results for failed playback on APPs, and not GitHub itself. Or posts where the commenter abandons the thread after being asked to specify what they are talking about.

Also here is a link to the issue I tried submitting, where I had to edit in a gfycat link to show the video.

https://github.com/K1rakishou/Kuroba-Experimental/issues/809

Edit: ALSO when I download the video I uploaded to GitHub and try to play it back locally, it still does not play.

**Comment by jfullstackdev** [comment]
seems the video itself is corrupted, even if I open that directly it should play because it's now just the Chrome and the video, but it's not playing

![image](https://user-images.githubusercontent.com/47092464/188314804-affd4c20-e950-4034-8df5-2f6b0a272cdd.png)

**Reply by reportthings** [reply]
Any idea why my videos corrupt when I upload them to GitHub? They work fine on gfycat, YouTube, etc.

**Reply by jfullstackdev** [reply]
I tried downloading it, it's not corrupted. Seems GitHub maybe has this bug by now.

**Reply by reportthings** [reply]
Strange. I downloaded it too and it is corrupted for me

**Comment by charbelnicolas** [comment]
This happens to me all the time... Any fix for this?

**Reply by reportthings** [reply]
Nope, this issue still exists 2 years later. Github basically told me to pound sand.

**Reply by fredrivett** [reply]
still seeing this for like 60% of video uploads, makes reviewing PR's a right pain

**Comment by Anupmqr** [comment]
Reduce video file size
